### DISCUSSION TITLE
Disappearing background image when alt-tabbing away

### DISCUSSION THREAD

**Discussion by zkwbbr** [tailwindlabs/tailwindcss#1646] [Help]
I am experiencing part of the background image disappear when I alt-tab away from the screen for a while. For example, I view the site for the first time with no problem, then I alt-tab away from the site for about 10-15mins. Now when I return to the site, part of the background is gone.

I noticed the background that disappears has an element "above" it that has `class="hidden"`, typically this is a nav menu in mobile breakpoint. 

It is quite hard to replicate since it only happens occasionally but I believe this is the part of the code where the issue happens.

https://jsfiddle.net/krgot8Lj/1/

Note: I stripped the code down to bare minimum. If the issue was to happen in that code, the upper portion of the background image would disappear when you alt-tab away for a while. Hope that make sense. Any suggestions how to prevent this?

**Comment by zkwbbr** [comment]
The issue is happening in JSFiddle too, here's a screenshot: https://imgur.com/pKnRGsU
You can see the background completely disappear after alt-tabbing away for a few mins.
Note: This is in Firefox.

**Comment by LeoniePhiline** [comment]
This sounds like a browser bug. Try reporting it at https://bugzilla.mozilla.org/ .

**Comment by zkwbbr** [comment]
Possible fix: I moved the background to a `div` instead of `body`. So far I have not seen the issue yet.
